CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,74 @@
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.10/docs/antrea-l7-network-policy.md) for more information about this feature.
+- Add SupportBundleCollection feature which enables a CRD API for Antrea to collect support bundle files on any K8s Node or ExternalNode, and upload to a user defined file server. ([#4184](https://github.com/antrea-io/antrea/pull/4184) [#4338](https://github.com/antrea-io/antrea/pull/4338) [#4249](https://github.com/antrea-io/antrea/pull/4249), [@wenyingd] [@mengdie-song] [@ceclinux])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.10/docs/support-bundle-guide.md) for more information about this feature.
+- Add support for NetworkPolicy for Cross-cluster Traffic. ([#4432](https://github.com/antrea-io/antrea/pull/4432) [#3914](https://github.com/antrea-io/antrea/pull/3914), [@Dyanngg] [@GraysonWu])
+    * Setting `scope` of an ingress peer to `clusterSet` expands the scope of the `podSelector` or `namespaceSelector` to the entire ClusterSet.
+    * Setting `scope` of `toServices` to `clusterSet` selects a Multi-cluster Service. ([#4397](https://github.com/antrea-io/antrea/pull/4397), [@Dyanngg])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.10/docs/multicluster/user-guide.md#networkpolicy-for-cross-cluster-traffic) for more information about this feature.
+- Add support for containerd runtime for antrea-agent on Windows. ([#4279](https://github.com/antrea-io/antrea/pull/4279), [@XinShuYang])
+- Add the following capabilities to the ExternalNode feature:
+    * Add the containerize option for antrea-agent installation on Linux VMs . ([#4413](https://github.com/antrea-io/antrea/pull/4413), [@Nithish555])
+    * Add support for RHEL OS. ([#4323](https://github.com/antrea-io/antrea/pull/4323), [@Nithish555])
+- Add [documentation](https://github.com/antrea-io/antrea/blob/release-1.10/docs/multicast-guide.md) for Antrea Multicast. ([#4339](https://github.com/antrea-io/antrea/pull/4339), [@ceclinux])
+
+### Changed
+
+- Extend `antctl mc get joinconfig` to print member token Secret. ([#4363](https://github.com/antrea-io/antrea/pull/4363), [@jianjuns])
+- Improve support for Egress in Traceflow. ([#3926](https://github.com/antrea-io/antrea/pull/3926), [@Atish-iaf])
+- Add Antrea agent configuration options to enable and configure secondary network specific OVS parameters. ([#4440](https://github.com/antrea-io/antrea/pull/4440), [@adwaitni])
+- Add NodePortLocalRange field for agentinfo. ([#4379](https://github.com/antrea-io/antrea/pull/4379), [@wenqiq])
+- Use format "namespace/name" as the key for ExternalNode span calculation. ([#4401](https://github.com/antrea-io/antrea/pull/4401), [@wenyingd])
+- Enclose Pod labels with single quote in s3 uploader for Snowflake use cases. ([#4334](https://github.com/antrea-io/antrea/pull/4334), [@dreamtalen])
```

Comment:
Suggestion:
> Enclose Pod labels with single quotes when uploading CSV record to S3 in the FlowAggregator.

It makes more sense to mention the FlowAggregator and not Snowflake, as the S3 uploader is not Snowflake-specific. All big data tools can usually consume data directly from S3.

Reply:
done